src/js/modules/infragistics.ui.combo.js

```diff
@@ -1965,9 +1965,31 @@
 			if ($.ig && $.ig.formatter) {
 				if (this.options.format === "auto" &&
 					($.type(item) === "date" || $.type(item) === "number")) {
-					item = $.ig.formatter(item, null, null);
+					item = $.ig.formatter(
```

Comment:
This seems like the better option, but we'll introduce a breaking change and we'll also have to change the [formatter topic](https://www.igniteui.com/help/formatting-dates-numbers-and-strings). If we are good with them, I'll refactor it and change the topic.